src/main/java/rest/EmployeeService.java

```diff
@@ -0,0 +1,10 @@
+package rest;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.List;
+
+public interface EmployeeService {
+    List<Employee> hiredEmployees() throws IOException;
+    BigInteger getSalary(Long id);
```

Comment:
I'm not sure about naming - whom salary this method return? can salary be like 123.123 ?